crates/client/src/api.rs

```diff
@@ -230,20 +230,23 @@ impl Client {
 
         let res = self.client.get(url).send().await?;
 
-        if res.status().is_success() {
-            if let Some(warg_url) = res
-                .json::<WellKnownConfig>()
-                .await
-                .map_err(|_| {
-                    ClientError::InvalidWellKnownConfig(self.url.registry_domain().to_string())
-                })?
-                .warg_url
-            {
-                Ok(Some(RegistryUrl::new(warg_url)?))
-            } else {
-                Ok(None)
-            }
+        if !res.status().is_success() {
+            tracing::debug!("the `.well-known` config was not found");
```

Comment:
Can we include the actual `res.status()` here? I'm mostly interested in this logging for surprising non-404 failures.

Reply:
Fixed